src/Saritasa.Tools.Messages/Common/Repositories/LogglyMessageRepository.cs

```diff
@@ -37,15 +45,29 @@ public class LogglyMessageRepository : IMessageRepository, IDisposable
         /// .ctor
         /// </summary>
         /// <param name="token">Customer token.</param>
-        public LogglyMessageRepository(string token)
+        /// <param name="username">Customer username.</param>
+        /// <param name="password">Customer password.</param>
+        /// <param name="accountDomain">Customer domain name</param>
+        public LogglyMessageRepository(string token, string username = "", string password = "", string accountDomain = "")
```

Comment:
Instead of have one universal .ctor make another one with token, username, password etc. So there should be 2 constructors.